src/test/java/utils/ScreenshotUtil.java

```diff
@@ -0,0 +1,19 @@
+package utils;
+
+import lombok.SneakyThrows;
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+
+import java.io.File;
+
+public class ScreenshotUtil {
+    @SneakyThrows
+    public static void getSnapShot(WebDriver webdriver, String fileWithPath) {
+        TakesScreenshot scrShot = ((TakesScreenshot) webdriver);
```

Comment:
Please make getSnapShot without parameters.
You implemented singleton pattern for web driver, so you can get driver directly in the method
FilePath - create in the method

Reply:
fixed